wave_utils/src/config.cpp

```diff
@@ -122,6 +122,7 @@ int ConfigParser::loadPrimitive(ConfigParam param) {
         case FLOAT: *param.f = node.as<float>(); break;
         case DOUBLE: *param.d = node.as<double>(); break;
         case STRING: *param.s = node.as<std::string>(); break;
+        default: return -3;
```

Comment:
I'll do another issue to write a doc for this soon. In the mean time let's just keep it this way for the PR? (else this goes beyond the scope of this PR)